### DISCUSSION TITLE
可以随意自定义箭头样式吗？

### DISCUSSION THREAD

**Discussion by zhangbaojia** [antvis/G6#6534] [Q&A]
我看API中箭头类型只能选择配置以下几种：
![image](https://github.com/user-attachments/assets/3b6dca3d-9c66-48b3-9655-a5feed6d3332)

可以自定义箭头吗？我需要这种一对多、多对一或者多对多的箭头
![image](https://github.com/user-attachments/assets/56b7886f-84f6-46fd-a28e-1be132fcb266)

**Comment by Rey-Wang** [comment]
自定义节点用 path 画一个
